modules/account-lib/src/coin/stx/utils.ts

```diff
@@ -435,6 +435,18 @@ export function stringifyCv(cv: ClarityValue): any {
     case ClarityType.Int:
     case ClarityType.UInt:
       return { type: cv.type, value: cv.value.toString() };
+    case ClarityType.OptionalSome:
+      return { type: cv.type, value: stringifyCv(cv.value) };
+    case ClarityType.Tuple:
+      return {
+        type: cv.type,
+        data: Object.assign({}, ...Object.entries(cv.data).map(([key, value]) => ({ [key]: stringifyCv(value) }))),
```

Comment:
This is a complicated way to do `_.mapValues()`, I would try to use that too instead (since you already have it in scope anyway)
```suggestion
        data: _.mapValues(cv.data, (value) => stringifyCV(value)),
```